src/core_plugins/kibana/public/dashboard/styles/index.less

```diff
@@ -13,13 +58,12 @@
 dashboard-grid {
   display: block;
   margin: 0;
-  padding: 5px;
+  padding: 20px;
```

Comment:
In general I've increased the padding on a bunch of our inner panels (see monitoring...etc) to give the controls some space. This was actually one of the few places I missed in my big K6 PR.